apps/freelan/config/freelan.cfg

```diff
@@ -416,18 +416,16 @@ public_endpoint=0.0.0.0
 # the netmask is determined according to the IPv4 class. It is recommended that
 # you set the network.enable_dhcp_proxy option.
 #
-# You may specify an empty value.
+# Commenting out, will result in no IPv4 networking. You cannot supply a blank value.
 #
-# Default: 9.0.0.1/24
 #ipv4_address_prefix_length=9.0.0.1/24
```

Comment:
A recent change that was made for freelan was that starting it with the default configuration "works" out-of-the-box.

Perhaps you could just uncomment this configuration so that this particular behaviour is preserved ?